web/theme.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:html';
+
+void init() {
+
+  var bodyElement = document.body;
+
+  if( bodyElement == null ){
+    return;
+  }
+
+  var theme = document.getElementById('theme') as InputElement;
+
+  theme.addEventListener('change', (event) {
```

Comment:
What does `theme.addEventListener('change'` do? This callback is called when?

Reply:
It's called every time the user changes the theme using the toggle input.

Reply:
Haha brain fart. I have not written web in a _long_ time. Thanks for bearing with me!